cmd/oras/root/pull.go

```diff
@@ -219,14 +228,16 @@ func doPull(ctx context.Context, src oras.ReadOnlyTarget, dst oras.GraphTarget,
 		var ret []ocispec.Descriptor
```

Comment:
We can change the above `nodes = append(nodes, *config)` to below
```golang
if config.Size != ocispec.DescriptorEmptyJSON.Size && config.Digest != ocispec.DescriptorEmptyJSON.Digest {
    nodes = append(nodes, *config)
}

```

Reply:
Added in https://github.com/oras-project/oras/pull/1164/commits/3ab0152c38be2bafac2cb985d8731a8a00d3ae54

Reply:
Do you want to send the PR over this @qweeah ?